player/Player.gd

```diff
@@ -1,18 +1,25 @@
 extends Area2D
 
 export var speed: float = 1024
+export var fireDelay: float = 0.16
+
 var velocity := Vector2(0, 0)
-onready var bullet_instance = preload("res://scenes/spaceship/bullet.tscn") #instancia do projetil
 var shootType = 1 
 
+onready var bullet_instance = preload("res://Bullet/Bullet.tscn") #instancia do projetil
```

Comment:
Here is returning loading error. I get the error:
Parser error: unable to preload resource at path: res://Bullet/Bullet.tscn

Reply:
Fixed by 245e2e5b72aecad99298956369184993a4b71ede